target/linux/ipq40xx/image/Makefile

```diff
@@ -347,24 +347,26 @@ TARGET_DEVICES += dlink_dap-2610
 define Device/edgecore_ecw5211
 	$(call Device/FitImage)
 	$(call Device/UbiFit)
-	DEVICE_TITLE := Edgecore ECW5211
+	DEVICE_VENDOR := Edgecore
+	DEVICE_MODEL := ECW5211
+	SOC := qcom-ipq4018
 	BLOCKSIZE := 128k
 	PAGESIZE := 2048
 	DEVICE_DTS_CONFIG := config@ap.dk01.1-c2
-	DEVICE_DTS := qcom-ipq4018-ecw5211
-	DEVICE_PACKAGES := ipq-wifi-edgecore_ecw5211 kmod-tpm-i2c-atmel kmod-usb-acm uboot-envtools
+	DEVICE_PACKAGES := kmod-tpm-i2c-atmel kmod-usb-acm uboot-envtools
 endef
 TARGET_DEVICES += edgecore_ecw5211
 
 define Device/edgecore_oap100
 	$(call Device/FitImage)
 	$(call Device/UbiFit)
-	DEVICE_TITLE := Edgecore OAP100
+	DEVICE_VENDOR := Edgecore
+	DEVICE_MODEL := OAP100
```

Comment:
I'd keep the OAP100 changes separate.

Reply:
Sure. This is still a draft, I'll clean it up in the next round.